tests/__init__.py

```diff
@@ -850,15 +853,24 @@ def wrapper(wrapped, instance, args, kwargs):
             ret = wrapped(*args, **kwargs)
```

Comment:
Can we include this in the below try-finally block in case some exception is raised in this call and goes uncaught?

Reply:
Ah yes, good catch.